openwisp_firmware_upgrader/api/tests/base/__init__.py

```diff
@@ -0,0 +1,41 @@
+from django.contrib.auth import get_user_model
+from django.test import Client
+from django.urls import reverse
+from openwisp_firmware_upgrader.tests.base import TestUpgraderMixin
+from swapper import load_model
+
+from openwisp_users.tests.utils import TestMultitenantAdminMixin
+
+BatchUpgradeOperation = load_model('firmware_upgrader', 'BatchUpgradeOperation')
+Build = load_model('firmware_upgrader', 'Build')
+Category = load_model('firmware_upgrader', 'Category')
+DeviceFirmware = load_model('firmware_upgrader', 'DeviceFirmware')
+FirmwareImage = load_model('firmware_upgrader', 'FirmwareImage')
+UpgradeOperation = load_model('firmware_upgrader', 'UpgradeOperation')
+
+
+user_model = get_user_model()
+
+
+class TestAPIUpgraderMixin(TestMultitenantAdminMixin, TestUpgraderMixin):
+    device_firmware_model = DeviceFirmware
+    upgrade_operation_model = UpgradeOperation
+    batch_upgrade_operation_model = BatchUpgradeOperation
+    firmware_image_model = FirmwareImage
+    build_model = Build
+    category_model = Category
```

Comment:
if it's a mixin, it can live also without the definitions of the models here, that definition has to be put in the test class which inherits the mixin